cmd/if-mover/main.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net"
+	"strconv"
+	"strings"
+
+	"github.com/vishvananda/netlink"
+	"github.com/vishvananda/netns"
+)
+
+const (
+	ifBase     = "mlink"
+	hostNsFile = "/tmp/hostns.txt"
+	multusLink = "net1"
+)
+
+func main() {
+
+	log.Println("Determining host namespace id")
+	// The Host network namespace name comes from a file generated by the following command:
+	// ip --json netns list-id | jq -j ".[] | select(.nsid == 0).name" > /tmp/hostns.txt
+	// It's expected that this was run before this program runs.
+	// This was the most consistent way I found to convey the host namespace to a golang program.
+
+	hostNsBytes, err := ioutil.ReadFile(hostNsFile)
```

Comment:
I think reading an env variable would be sufficient `ioutil.ReadFile` seems a bit overkill here.